kernel/modops.ml

```diff
@@ -339,7 +339,8 @@ let strengthen_const mp_from l cb resolver =
       | Polymorphic_const ctx -> Univ.make_abstract_instance ctx
     in
       { cb with
-	const_body = Def (Mod_subst.from_val (mkConstU (con,u)));
+        const_body = Def (Mod_subst.from_val (mkConstU (con,u)));
+        const_private_poly_univs = None;
```

Comment:
The checker caught the need for this in HoTT, so I fixed it. I added an Include to the new test file but it will have to wait for #8655 to be actually checked.

Reply:
That looks right, it is actually changing the implementation of cb in the module signature with an alias to cb in the module implementation which itself holds the private universes.